gensim/models/translation_matrix.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python
+# encoding: utf-8
+import warnings
+import numpy as np
+import random
+
+from collections import OrderedDict
+from gensim import utils
+from six import string_types
+
+random.seed(2333)
```

Comment:
Please don't hardcode global random seed. You should create personal `Random` object and use it